stable/aws-pod-identity-webhook/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for the Amazon EKS Pod Identity Webhook
+name: aws-pod-identity-webhook
+version: 0.1.0
+icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
+home: https://github.com/aws/amazon-eks-pod-identity-webhook
+sources:
+- https://github.com/aws/amazon-eks-pod-identity-webhook
+maintainers:
```

Comment:
Can you add me as well here?

Reply:
I had you there in the beginning but read the message here: https://github.com/helm/charts/pull/17099#issuecomment-530873181

So I think we just have @micahhausler there for now as they are already a member of the kubernetes org.

And then you guys can create a PR later to add yourself when you become a [trusted-collaborator](https://github.com/helm/charts#trusted-collaborator)